### PR TITLE
map2reg: combine regions using CXCRegion instead of dumping to ascii file

### PR DESCRIPTION
**Pull request by kglotfelty**
Unfortunately need to bite the bullet and combine regions using more memory intensive way.  Reading ASCII region with many sided polygon fails.

### CODE REVIEW

File: bin/map2reg [hunk 1]
```diff
@@ -1,6 +1,6 @@
-#!/usr/bin/env python
+#!/soft/miniconda/envs/contrib_test/bin/python3.11
```
Comment (DougBurke) [inline bin/map2reg:1]: We don't want this change

File: bin/map2reg [hunk 2]
```diff
@@ -136,20 +140,12 @@ def main():
...
-    # an ASCII region file and then reading back in as a single
-    # CXCRegion is much faster
-
-    # Combine into single region file
-    tmpfile = NamedTemporaryFile(dir=os.environ["ASCDS_WORK_PATH"],
-                                 suffix=".reg", delete=True)
-    with open(tmpfile.name, "w") as outfile:
-        for myreg in outreg:
-            outfile.write(myreg+"\n")
+    # Unfortunately writing all out to an ascii file sometimes fails
```
Comment (DougBurke) [inline bin/map2reg:143]: I assume there's no easy (or maybe any) way to tell when the dump to an ASCII file approach will work?

Comment (kglotfelty) [reply bin/map2reg:143]: There's probably a per-line string length defined somewhere in the DM (it parses the `region()` syntax and sends lines to region lib).  Probably some variation of n*1024 +/- m depending on how new-line/string terminator are handled.

File: bin/map2reg (hunk 1 above)
Comment (kglotfelty) [reply bin/map2reg]: fixed

**Pull request merged**